src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -55,12 +55,14 @@ export class PartCreationComponent implements OnInit, OnDestroy {
      * they need common data so that the parent calculates/retrieves the data then share it
      * with the part creation component
      */
-    public static VERBOSE: boolean = false;
+    public static VERBOSE: boolean = true;
+
+    public static TOKEN_INTERVAL: number = 5 * 1000;
 
     public partType: typeof PartType = PartType;
 
     @Input() partId: string;
-    @Input() userName: string;
+    @Input() authUser: AuthUser;
```

Comment:
note: dans AuthService, tu as accès à .user et .uid qui sont sûr d'avoir une valeur pour toute page accessible uniquement aux utilisateurs vérifiés. Ici c'est le cas, donc tu peux directement faire `this.authService.user.get()` pour accéder à l'utilisateur, sand demander d'input supplémentaire